index.bs

```diff
@@ -8171,6 +8217,75 @@ The [=remote end steps=] are:
  1. Return [=success=].
 
+## <dfn>Set Credential Properties</dfn> ## {#sctn-automation-set-credential-properties}
+
+The [=Set Credential Properties=] [=extension command=] allows setting the |backupEligibility| and |backupState| [=credential properties=] of
+a [=Virtual Authenticator=]'s [=public key credential source=]. It is defined as follows:
+
+<figure id="table-setFlags" class="table">
+    <table class="data">
+        <thead>
+            <tr>
+                <th>HTTP Method</th>
+                <th>URI Template</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>POST</td>
+                <td>`/session/{session id}/webauthn/authenticator/{authenticatorId}/credentials/{credentialId}/props`</td>
+            </tr>
+        </tbody>
+    </table>
+</figure>
+
+The <dfn>Set Credential Properties Parameters</dfn> is a JSON [=Object=] passed to the [=remote end steps=] as |parameters|.
+It contains the following |key| and |value| pairs:
+
+<figure id="table-setCredentialPropertiesParameters" class="table">
+    <table class="data">
+        <thead>
+            <tr>
+                <th>Key</th>
+                <th>Description</th>
+                <th>Value Type</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>|backupEligibility|</td>
+                <td>The [=backup eligibility=] [=credential property=].</td>
+                <td>boolean</td>
+            </tr>
+            <tr>
+                <td>|backupState|</td>
+                <td>The [=backup state=] [=credential property=].</td>
+                <td>boolean</td>
+            </tr>
+        </tbody>
+    </table>
+</figure>
+
+The [=remote end steps=] are:
+
+ 1. If |parameters| is not a JSON [=Object=], return a [=WebDriver error=] with [=WebDriver error code=]
+     [=invalid argument=].
+
+     Note: |parameters| is a [=Set Credential Properties Parameters=] object.
+ 1. If |authenticatorId| does not match any [=Virtual Authenticator=] stored in the [=Virtual Authenticator
+     Database=], return a [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+ 1. Let |credential| be the [=public key credential source=] managed by |authenticator| matched by |credentialId|.
+ 1. If |credential| is empty, return a [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+ 1. Let |backupEligibility| be the |parameters|' |backupEligibility| property.
+ 1. If |backupEligibility| is defined, set the [=backup eligibility=] [=credential property=] of |credential| to its value.
+
+    Note: Normally, the |backupEligibility| property is permanent to a [=public key credential source=].
+    [=Set Credential Properties=] allows changing it for testing and debugging purposes.
+
+ 1. Let |backupState| be the |parameters|' |backupState| property.
+ 1. If |backupState| is defined, set the [=backup state=] [=credential property=] of |credential| to its value.
```

Comment:
I think it's probably worth being a bit more explicit here, to eliminate any doubt of whether "it" refers to `|backupEligibility|` or to `the [=backup eligibility=] [=credential property=] of |credential|`.

```suggestion
 1. If |backupEligibility| is defined, set the [=backup eligibility=] [=credential property=] of |credential| to the value of |backupEligibility|.

    Note: Normally, the |backupEligibility| property is permanent to a [=public key credential source=].
    [=Set Credential Properties=] allows changing it for testing and debugging purposes.

 1. Let |backupState| be the |parameters|' |backupState| property.
 1. If |backupState| is defined, set the [=backup state=] [=credential property=] of |credential| to the value of |backupState|.
```